crates/sui-bridge/src/sui_client.rs

```diff
@@ -258,8 +305,28 @@ impl SuiClientInner for SuiSdkClient {
             .await
     }
 
-    async fn get_bridge_committee(&self) -> Result<BridgeCommittee, Self::Error> {
-        unimplemented!()
+    async fn get_bridge_committee(&self) -> Result<MoveTypeBridgeCommittee, Self::Error> {
+        let object_id = *get_bridge_object_id();
+        let resp = self
+            .read_api()
+            .get_object_with_options(object_id, SuiObjectDataOptions::default().with_bcs())
+            .await?;
+        if resp.error.is_some() {
+            return Err(Self::Error::DataError(format!(
+                "Can't get bridge object {:?}: {:?}",
+                object_id, resp.error
+            )));
+        }
+        let move_object = resp
+            .data
+            .unwrap() // unwrap: Bridge object must exist
+            .bcs
+            .unwrap(); // unwrap requested bcs data
+                       // unwrap: Bridge object must be a Move object
+        let bcs = move_object.try_as_move().unwrap();
+        let bridge_dynamic_field: Field<u64, MoveTypeBridgeInner> =
+            bcs::from_bytes(&bcs.bcs_bytes)?;
```

Comment:
I don't understand this part. The object you get here is the bridge object itself, right?

Reply:
nit: you can use to_rust() on resp.
Also please add TODOs at the top of this function as all these logic will eventually move to fullnode